src/commands/hosting-clone.ts

```diff
@@ -74,7 +77,27 @@ export default new Command("hosting:clone <source> <targetChannel>")
         )}, creating it...`
       );
       tChannel = await createChannel("-", targetSiteId, targetChannelId);
+      if (!tChannel) {
```

Comment:
If it cannot create the channel, does `createChannel` return a falsey value or does it throw an error? (my guess in the second, so this would need to be a `catch`)